tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -15,6 +15,7 @@
     ['window-close-v1', SawyerWindowCloseV2Policy(), .1, 0.44, {}],
     ['window-close-v2', SawyerWindowCloseV2Policy(), 0., 0.98, {}],
     ['window-close-v2', SawyerWindowCloseV2Policy(), .1, 0.98, {}],
+    ['button-press-v1', SawyerButtonPressV1Policy(), .1, 0.95, {}],
```

Comment:
What's the accuracy without adding noise?

Reply:
~95% with noise
~98% without noise

Reply:
That might be something to make note of -- 3% is a higher discrepancy than most of the other policies